yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
empty file here?

Reply:
This is a semi-hack for the CI tests `actions/setup-node@v2` runs `yarn install` in the project root and throws a frozen lockfile error if this is removed.